src/SprykerEco/Yves/Computop/Mapper/Init/PrePlace/AbstractPrePlaceMapper.php

```diff
@@ -69,15 +75,22 @@ public function createComputopPaymentTransfer(TransferInterface $quoteTransfer)
      * @param string $merchantId
      * @param string $data
      * @param int $length
+     * @param string $urlBack
      *
      * @return string
      */
-    protected function getUrlToComputop($merchantId, $data, $length)
+    protected function getUrlToComputop($merchantId, $data, $length, $urlBack = '')
     {
-        return $this->getActionUrl() . '?' . http_build_query([
-                ComputopApiConfig::MERCHANT_ID => $merchantId,
-                ComputopApiConfig::DATA => $data,
-                ComputopApiConfig::LENGTH => $length,
-            ]);
+        $queryData = [
+            ComputopApiConfig::MERCHANT_ID => $merchantId,
+            ComputopApiConfig::DATA => $data,
+            ComputopApiConfig::LENGTH => $length,
+        ];
+        
+        if ($urlBack !== '') {
```

Comment:
Will it be added to all queries? Because we need this param only for Credit Card

Reply:
Seems so, only for CC. Could be moved to overridden method in CreditCardMapper then?

Reply:
I think it would be better